build.sh

```diff
@@ -55,8 +55,8 @@ do
 	if grep -q "netcoreapp1.0" "$testProject"; then
 		pushd $testDir
 
-		echo "$DOTNET test $testDir --configuration release --framework netcoreapp1.0"
```

Comment:
Shouldn't `build.sh` on Mac have `-notrait Platform=Linux`? And similar for `build.sh` on Linux?